package.json

```diff
@@ -25,21 +31,32 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react_ujs": "^2.6.1",
+    "shakapacker": "6.5.4",
     "stimulus": "^2.0.0",
-    "turbolinks": "^5.2.0"
+    "terser-webpack-plugin": "5",
+    "turbolinks": "^5.2.0",
+    "webpack": "5",
+    "webpack-assets-manifest": "5",
+    "webpack-cli": "4",
+    "webpack-dev-server": "^4.11.1",
+    "webpack-merge": "5"
   },
-  "version": "0.1.0",
   "devDependencies": {
     "babel-eslint": "^10.1.0",
     "eslint": "^7.2.0",
     "eslint-config-airbnb": "18.2.1",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.21.5",
-    "eslint-plugin-react-hooks": "^1.7.0",
-    "webpack-dev-server": "^3.11.2"
+    "eslint-plugin-react-hooks": "^1.7.0"
+  },
+  "version": "0.1.0",
+  "babel": {
+    "presets": [
+      "./node_modules/shakapacker/package/babel/preset.js"
+    ]
   },
-  "resolutions": {
-    "@psychobolt/react-paperjs": "1.0.0"
-  }
```

Comment:
why does this go away?

Reply:
Because yarn is able to resolve this to 1.0.0 without the hint.

Reply:
I've readded this. I'm not sure if this is fixed now that 1.0.3 is out.